fog/api/proto/view.proto

```diff
@@ -204,20 +204,6 @@ enum TxOutSearchResultCode {
     RateLimited = 5;
 }
 
-
-/// A Redacted Fog Transaction Output.
-/// This is the same as a normal TxOut, except that the fog hint is removed after processing, to save storage.
-message FogTxOut {
-    /// Amount.
-    external.Amount amount = 1;
-
-    /// Public key.
-    external.CompressedRistretto target_key = 2;
-
-    /// Public key.
-    external.CompressedRistretto public_key = 3;
-}
-
```

Comment:
this is omitted because it's not actually used in any public facing APIs, kyle and alex thought it's better to remove it if that isn't the case since it might confuse someone